### PR TITLE
Update quest_poi_points.md

### PR DESCRIPTION
**Pull request by Stifler82**
More details on the arrangement of coordinates.

### CODE REVIEW

File: docs/quest_poi_points.md [hunk 1]
```diff
@@ -49,6 +49,11 @@ Example quest: Secreat Communication.
 | 8318    | 3    | 6    | -6483 | -217 | 0             |
 | 8318    | 3    | 7    | -6326 | -7   | 0             |
 
+**These points are all the little corners on the blue box. idx1 represents an area defined by the points of idx2 to draw the shape.**
```
Comment (Kitzunu) [inline docs/quest_poi_points.md:52]: This is already described above

> The actual points are the corners of each polygon.

File: docs/quest_poi_points.md [hunk 2]
```diff
@@ -49,6 +49,11 @@ Example quest: Secreat Communication.
...
+![image](https://user-images.githubusercontent.com/1884642/119476187-bca11b00-bd45-11eb-95e5-876960f24457.png)
+
+
+
```
Comment (Kitzunu) [inline docs/quest_poi_points.md:56]: Only 1 space needed

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: docs/quest_poi_points.md (hunk 1 above)
Comment (Stifler82) [reply docs/quest_poi_points.md:52]: Yes I understand, but this description helps to better understand if associated with the image.

**Review by Kitzunu — APPROVED** [review]
Fine for now, let's merge before I change my mind =P

**Comment by Stifler82** [comment]
Why is it so @Kitzunu ?
what did i do wrong?
I would like the picture below the description.
image

**Comment by Kitzunu** [comment]
Intresting, probably because Markdown is ass. I can take care of it

**Comment by Stifler82** [comment]
> Intresting, probably because Markdown is ass. I can take care of it

ty @Kitzunu

**Pull request merged**